src/components/Select/Select.js

```diff
@@ -122,12 +115,34 @@ export const Select = ({
 
   const SelectTag = getTag()
 
+  const getClasses = () => {
+    const cName = className || ''
+    const cOverride = classOverrides || ''
+    const compactClass = isCompact ? styles.Compact : ''
+    return `${cName} ${compactClass} ${styles.root} ${cOverride}`
+  }
+
   return (
-    <div className={wrapperClass} data-tid={rest['data-tid']}>
+    <div className={title ? styles.wrapper : ''} data-tid={rest['data-tid']}>
```

Comment:
If it makes things more "readable" I really appreciate the pattern where you do: `const wrapper = <build the styles string>` but here i think it doesn't help too much.